src/org/loklak/api/search/SearchServlet.java

```diff
@@ -317,10 +320,10 @@ public void run() {
                 // we silently do nothing here and return what we listed so far
             }
             m.put("statuses", statuses);
-            
```

Comment:
Please revert this line, it's not part of the issue. If you want to do some house-keeping or cleaning, please create a new PR and Issue for it.